pkg/logql/evaluator.go

```diff
@@ -31,14 +31,15 @@ var (
 
 // Params details the parameters associated with a loki request
 type Params interface {
-	Query() string
+	QueryString() string
 	Start() time.Time
 	End() time.Time
 	Step() time.Duration
 	Interval() time.Duration
 	Limit() uint32
 	Direction() logproto.Direction
 	Shards() []string
+	GetExpression() syntax.Expr // TODO: rename to GetPlan
```

Comment:
TODO

Reply:
This was actually a question :slightly_smiling_face: Would you prefer `GetPlan` or `GetExpression`?

Reply:
I like `GetExpression` more.

Reply:
Ditto